src/bounded_loop.rs

```diff
@@ -0,0 +1,156 @@
+use std::fmt;
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::env;
+use std::collections::HashMap;
+use std::fs::OpenOptions;
+use std::io::Write;
+use std::io::{BufReader, BufRead, Read};
+use std::fs::File;
+
+lazy_static!{
+    static ref MUTAGEN_LOOP_COUNT: HashMap<LoopId, usize> = {
+        let mut h = HashMap::new();
+
+        File::open("target/mutagen/loops.txt").map(|mut file| {
```

Comment:
I was thinking about doing some abstraction for IPC, as there's some repeated code with the coverage communication, but I postponed this, as I wanted to finish this task first.

Reply:
I think [men_file](https://docs.rs/men_file) could work well for us.